src/x11-utils.cpp

```diff
@@ -6,18 +6,19 @@
 
 #include "globals.h"
 
+using std::vector;
+
 /**
  * \brief   cut a rect out of the window, s.t. the window has geometry rect and
```

Comment:
This comment is here and in the .h file and needs to be updated, too

Reply:
[x] fixed.

Reply:
Thanks!